pkg/dba/dba.go

```diff
@@ -35,6 +37,8 @@ func NewDefaultOptions() Options {
 		AnalyzeTimeout: 10 * time.Minute,
 		PostAnalyze:    true,
 
+		BloatQueryTimeout: 5 * time.Minute,
```

Comment:
Maybe default this to 30 seconds to keep it passive?

Reply:
Good idea